performance/db/mongo-index-test.py

```
@@ -0,0 +1,64 @@
+    lantern_id = f"{DEFAULT_USER_NAME}-{i}"
+    start = time.perf_counter()
+    explain = collection.find({"lantern_id": lantern_id}).explain()
+    end = time.perf_counter()
+
+    execution_time = (end - start) * 1000  # ms
+    execution_times.append(execution_time)
+
+    try:
+        stats = explain["executionStats"]
+        doc_counts.append(stats.get("totalDocsExamined", -1))
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)